cluster-operator/src/test/java/io/strimzi/operator/cluster/model/CruiseControlTest.java

```diff
@@ -379,6 +380,45 @@ public void testImagePullPolicy() {
         assertThat(ccContainer.getImagePullPolicy(), is(ImagePullPolicy.IFNOTPRESENT.toString()));
     }
 
+    @ParallelTest
+    public void testCpuUtilization() {
+        Kafka k = createKafka(cruiseControlSpec);
+        ResourceRequirements resources = new ResourceRequirementsBuilder()
+                .withLimits(singletonMap("cpu", new Quantity("800m"))) //800m = 80%
+                .build();
+        k.getSpec().getKafka().setResources(resources);
+
+        Capacity generatedCapacity = new Capacity(k.getSpec());
+        assertThat(getCapacityConfigurationFromEnvVar(k, ENV_VAR_BROKER_CPU_UTILIZATION_CAPACITY), is(Integer.toString(generatedCapacity.getCpuUtilization())));
+        assertThat(getCapacityConfigurationFromEnvVar(k, ENV_VAR_BROKER_CPU_UTILIZATION_CAPACITY), is(Integer.toString(80)));
+
+        // default value
+        k.getSpec().getKafka().setResources(null);
+        generatedCapacity = new Capacity(k.getSpec());
+        assertThat(getCapacityConfigurationFromEnvVar(k, ENV_VAR_BROKER_CPU_UTILIZATION_CAPACITY), is(Integer.toString(generatedCapacity.getCpuUtilization())));
+        assertThat(getCapacityConfigurationFromEnvVar(k, ENV_VAR_BROKER_CPU_UTILIZATION_CAPACITY), is(Integer.toString(DEFAULT_BROKER_CPU_UTILIZATION_CAPACITY)));
+
+        // value without m
+        k = createKafka(cruiseControlSpec);
+        resources = new ResourceRequirementsBuilder()
+                .withLimits(singletonMap("cpu", new Quantity("2")))
+                .build();
+        k.getSpec().getKafka().setResources(resources);
+        generatedCapacity = new Capacity(k.getSpec());
+        assertThat(getCapacityConfigurationFromEnvVar(k, ENV_VAR_BROKER_CPU_UTILIZATION_CAPACITY), is(Integer.toString(generatedCapacity.getCpuUtilization())));
+        assertThat(getCapacityConfigurationFromEnvVar(k, ENV_VAR_BROKER_CPU_UTILIZATION_CAPACITY), is(Integer.toString(200)));
```

Comment:
We cannot specify percentage values outside  the interval [0,100] or else Cruise Control will crash when it reads the generated capacity config file we generate.